ui/packages/components/src/types/features.ts

```diff
@@ -0,0 +1,3 @@
+export type Features = {
+  history: number;
```

Comment:
Is this a number?

Reply:
Yea:
https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/%5BenvironmentSlug%5D/functions/%5Bslug%5D/runs/TimeFilter.tsx#L39-L41